solidity/contracts/libraries/staking/MinimumStakeSchedule.sol

```diff
@@ -9,12 +9,20 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 library MinimumStakeSchedule {
     using SafeMath for uint256;
 
+    // Apr-28-2020 02:52:46 AM UTC when KEEP token has been deployed
+    // TX:  0xea22d72bc7de4c82798df7194734024a1f2fd57b173d0e065864ff4e9d3dc014
+    uint256 public constant scheduleStart = 1588042366;
```

Comment:
Triple-check me please:

```
$ date -r 1588042366
Tue 28 Apr 2020 04:52:46 CEST
```

https://etherscan.io/tx/0xea22d72bc7de4c82798df7194734024a1f2fd57b173d0e065864ff4e9d3dc014

Reply:
04:52:46 CEST - I remember I observed sunshine back then so we should be good here 😆

Reply:
have the same:
```
date -r 1588042366
Tue Apr 28 04:52:46 CEST 2020
```
Tue Apr 28 04:52:46 CEST 2020 == Apr-28-2020 02:52:46 AM +UTC

Reply:
Confirmed
```
$ date -u -r 1588042366
Tue Apr 28 02:52:46 UTC 2020
```